src/swipeable.js

```diff
@@ -349,6 +358,12 @@ class Swipeable extends Component {
     this.content = node;
   }
 
+  applyDisplayRuleToChildNodes() {
+    for (let i = 0; i < this.content.childNodes.length; i++) {
+      this.content.childNodes[i].style.display = 'inline-block';
```

Comment:
I don't really understand why this is necessary, what property am I supposed to check for?